### ISSUE TITLE
LibreJS compliance

### ISSUE THREAD

**Issue by pizzamaker** [creativecommons/creativecommons.org#189]
As of now, GNU LibreJS warns of nonfree JavaScript on the website. This might be a false positive, but in any case it can be made compatible with the addon

Here's a few useful links to fix this bug:

* https://www.gnu.org/philosophy/javascript-trap.html
* https://www.fsf.org/blogs/licensing/rel-jslicense

Here's the LibreJS output:

    List of accepted JavaScript in https://creativecommons.org/
    
        This script is detected as free
        https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js

        List of blocked JavaScript in https://creativecommons.org/

         This script is detected as inline, nonfree, defining functions or methods, and the rest of the page as loading external scripts

        /* <![CDATA[ */
        var auth = {"wp_login_url":"http:\/\/creativecommons.org\/wordpress\/wp-login.php?redirect_to=%2F","public_warning":""};
        /* ]]> */
    
       

        NONTRIVIAL: an open method similar to xhr.open is used

    	  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    	  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    	  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    	  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

    	  ga('create', 'UA-2010376-1', 'auto');
    	  ga('send', 'pageview');

    	

    Whitelist

    This script is detected as inline, nonfree, defining functions or methods, and the rest of the page as loading external scripts

    $(document).ready(function() {

    if ($(window).width() > 800) {

    var strSorryIfyouTryToDebugThis="";
    strSorryIfyouTryToDebugThis += "<div id=\"takeover\" style=\"background-color: #dc642f;  position: absolute; top: 0; left: 0; width: 100% !important; z-index: 999; overflow: hidden;\">";
    strSorryIfyouTryToDebugThis += "<div style=\"background: white; padding: 1em;\"><div style=\"width: 100%; max-width: 1100px; margin: 0 auto;\"><a href=\"#\" id=\"takeover-link\" style=\"padding: 1em;\">Continue to Creative Commons<\/a><\/div><\/div>";
    strSorryIfyouTryToDebugThis += "<div class=\"container\" style=\"width: 100%; max-width: 1100px; margin: 0 auto;\">";
    strSorryIfyouTryToDebugThis += "<div class=\"row\">";
    strSorryIfyouTryToDebugThis += "<div class=\"col-md-6 col-md-offset-1 col-sm-12 col-xs-12\" style=\"height: 588px;\">";
    strSorryIfyouTryToDebugThis += "";
    strSorryIfyouTryToDebugThis += "<div  style=\"position: relative;";
    strSorryIfyouTryToDebugThis += "  top: 8%;\">";
    strSorryIfyouTry…

    WhitelistNONTRIVIAL: an open method similar to xhr.open is used
    https://creativecommons.org/wp-content/themes/creativecommons.org/js/libs/modernizr-2.0.6.min.js
    WhitelistThis script is detected as nonfree, external, and as defining functions or methods
    http://creativecommons.org/wp-content/plugins/authorizer/js/authorizer-public.js?ver=2.3.2
    WhitelistThis script is detected as nonfree, external, and as defining functions or methods
    http://creativecommons.org/wordpress/wp-includes/js/wp-embed.js?ver=4.4
    WhitelistNONTRIVIAL: Creates an xhr object
    https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js

Web Labels pages being used for this session

    https://creativecommons.org/wp-content/themes/creativecommons.org/weblabels.html

**Comment by pizzamaker** [comment]
Hi @mattl, there's no interest in fixing this?

**Comment by mattl** [comment]
Not really... There's Google Analytics, which isn't going anywhere, and some temporary pop up we're getting rid of in the new year. I think everything else is listed in our web labels page.
